app/src/main/java/com/example/quizacademytask/MainActivity.kt

```diff
@@ -15,9 +14,7 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(binding.root)
-
-        //Initiate the database
-        AppDatabase.getInstance(this@MainActivity)
+        deleteDatabase("database")
```

Comment:
this is confusing. Is this fuction does what it says? Are you deleting the database each time MainActivity is created?

Reply:
Sorry, this was definitely not supposed to be in the commit